src/test/java/seedu/jarvis/model/cca/CcaTrackerTest.java

```diff
@@ -2,15 +2,32 @@
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static seedu.jarvis.testutil.cca.TypicalCcas.CANOEING;
+import static seedu.jarvis.testutil.cca.TypicalCcas.GUITAR_ENSEMBLE;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class CcaTrackerTest {
 
-    private final CcaTracker ccaTracker = new CcaTracker();
+    private CcaTracker ccaTracker;
+
+    @BeforeEach
+    public void setUp() {
+        // TODO is this the right way of stubbing? Do I need to stub the internalCcaList too?
```

Comment:
try to address all your TODOs before making a PR to the team repo :) 
Alternatively u can make a static inner class `CcaStub` that extends `Cca` in your `CcaTrackerTest`

Reply:
ok